Content.Server/Crayon/CrayonSystem.cs

```diff
@@ -68,7 +70,7 @@ private void OnCrayonAfterInteract(EntityUid uid, CrayonComponent component, Aft
             return;
 
         if (component.UseSound != null)
-            SoundSystem.Play(component.UseSound.GetSound(), Filter.Pvs(uid), uid, AudioHelpers.WithVariation(0.125f));
+            _audio.PlayPvs(_audio.GetSound(component.UseSound), uid, AudioHelpers.WithVariation(0.125f));
```

Comment:
```suggestion
            _audio.PlayPvs(component.UseSound, uid, AudioHelpers.WithVariation(0.125f));
```
There's a different method that's also named `PlayPvs` that just takes a sound specifier instead of a string.